modules/evs/fsw/inc/cfe_evs_msg.h

```diff
@@ -0,0 +1,1287 @@
+**       result may be saving valuable event data.  However, inappropriately
+**       setting the log mode could result in a loss of critical information.
+**      Note: the event log is a back-up log to the on-board recorder.
+**
+**  \sa #CFE_EVS_WRITE_LOG_DATA_FILE_CC, #CFE_EVS_CLEAR_LOG_CC
+*/
+#define CFE_EVS_SET_LOG_MODE_CC 19
+
+/** \cfeevscmd Clear Event Log
+**
+**  \par Description
+**       This command clears the contents of the local event log.
+**
+**  \cfecmdmnemonic \EVS_CLRLOG
+**
+**  \par Command Structure
+**       #CFE_EVS_ClearLogCmd_t
+**
+**  \par Command Verification
+**       Successful execution of this command may be verified with
+**       the following telemetry:
+**       - \b \c \EVS_CMDPC - command execution counter will
+**       increment
+**
+**  \par Error Conditions
+**      This command may fail for the following reason(s):
+**      - Invalid SB message (command) length
+**
+**       Evidence of failure may be found in the following telemetry:
+**       - \b \c \EVS_CMDEC - command error counter will increment
+**       - An Error specific event message
+**
+**  \par Criticality
+**       Clearing the local event log is not particularly hazardous, as the
+**      result may be making available space to record valuable event data.
+**      However, inappropriately clearing the local event log could result
+**      in a loss of critical information.  Note: the event log is a back-up
+**      log to the on-board recorder.
+**
+**  \sa #CFE_EVS_WRITE_LOG_DATA_FILE_CC, #CFE_EVS_SET_LOG_MODE_CC
+*/
+#define CFE_EVS_CLEAR_LOG_CC 20
+/** \} */
+
+/* Event Type bit masks */
+#define CFE_EVS_DEBUG_BIT       0x0001
+#define CFE_EVS_INFORMATION_BIT 0x0002
+#define CFE_EVS_ERROR_BIT       0x0004
+#define CFE_EVS_CRITICAL_BIT    0x0008
+
+/* Output Port bit masks */
+#define CFE_EVS_PORT1_BIT 0x0001
+#define CFE_EVS_PORT2_BIT 0x0002
+#define CFE_EVS_PORT3_BIT 0x0004
+#define CFE_EVS_PORT4_BIT 0x0008
+
+/* EVS Log Modes */
+#define CFE_EVS_LOG_OVERWRITE 0
+#define CFE_EVS_LOG_DISCARD   1
+
+/******************  Structure Definitions *********************/
+
+/**
+** \brief Command with no additional arguments
+**/
+typedef struct CFE_EVS_NoArgsCmd
+{
+    CFE_MSG_CommandHeader_t CmdHeader; /**< \brief Command header */
+} CFE_EVS_NoArgsCmd_t;
+
+/*
+ * The NoArgsCmd is shared by several EVS command handlers.
+ * Create a unique type for each one so the prototypes will follow the naming pattern,
+ * allowing each command to evolve independently.
+ */
+typedef CFE_EVS_NoArgsCmd_t CFE_EVS_NoopCmd_t;
+typedef CFE_EVS_NoArgsCmd_t CFE_EVS_ResetCountersCmd_t;
+typedef CFE_EVS_NoArgsCmd_t CFE_EVS_ClearLogCmd_t;
+
+/**
+** \brief Write Event Log to File Command Payload
+**
+** For command details, see #CFE_EVS_WRITE_LOG_DATA_FILE_CC
+**
+**/
+typedef struct CFE_EVS_LogFileCmd_Payload
+{
+    char LogFilename[CFE_MISSION_MAX_PATH_LEN]; /**< \brief Filename where log data is to be written */
```

Comment:
Seems like unneeded structure to just hold char variable array, is the intent to have other information put here (ie future work?)